src/components/common/index.js

```diff
@@ -1,2 +1,26 @@
+import CustomButton from "./CustomButton.vue";
+import TabHeader from "./TabHeader.vue";
 import NavBar from "./NavBar.vue";
-export { NavBar };
+import SectionTitle from "./SectionTitle.vue";
+import SectionSubtitle from "./SectionSubtitle.vue";
+import CapitalLink from "./CapitalLink.vue";
+import CardTitle from "./CardTitle.vue";
+import CardSubtitle from "./CardSubtitle.vue";
+import CustomChip from "./CustomChip.vue";
+import MovieCard from "./MovieCard.vue";
+import MovieCardDetailed from "./MovieCardDetailed.vue";
+import MovieList from "./MovieList.vue";
+export {
```

Comment:
I don't think this is necessary actually 😄 
Doing just the following will work fine regardless, no need for big index files everywhere 😄 

```js
import NavBar from '@/components/common/NavBar.vue';
```